packages/mui-material/src/MenuItem/MenuItem.test.js

```diff
@@ -162,4 +169,26 @@ describe('<MenuItem />', () => {
       expect(context).to.have.property('dense', true);
     });
   });
+
+  describe('consume value Prop', () => {
+    const mount = createMount();
+
+    it('should accept primitive value', () => {
+      const wrapper = mount(<MenuItem value={123} />);
+
+      expect(wrapper.find(MenuItem).props().value).to.equal(123);
```

Comment:
Thanks for the unit tests, but let's not use the enzyme way but select the `MenuItem` by it's role `menuitem`. You can refer the other tests.

Reply:
@ZeeshanTamboli done.